### ISSUE TITLE
Configuration of MP4ThumbsAndGif plugin

### ISSUE THREAD

**Issue by flochman** [DanielnetoDotCom/YouPHPTube#268]
I bought "MP4ThumbsAndGif" plugin and installed it on 192.168.0.71/ph  site which is streaming application YouPHPTube.

I also installed YouPHPTube-Encoding application on 192.168.0.71/en

But I can not see the thumbnails on videos in streaming application. I see still the chaotic running image or like hollywood films image on video view.

What should be wrong? It seems that plugin does not works or encoding dos not works. How to config right them?

Thanks.

**Comment by DanielnetoDotCom** [comment]
Hi 

Is your encoder working?
Did you changed your encoder url on the streamer configurations menu?
Since you are in a local network you will not be able to use public encoder

**Comment by flochman** [comment]
Yes, on streamer configuration I have encoder setup to http://192.168.0.71/en

How I check that  is encoder working?

**Comment by DanielnetoDotCom** [comment]
can you download an youtube video on encoder?
can you upload a video on encoder?
If yes, you should be all set.

**Comment by flochman** [comment]
OK. I uploaded video in encoding application and yes, the queue was created the video was encoded and was uploaded to stream application and thumbnails was created. So plugin works.

So it seems all work.

But I uploaded video from encoding application.

I would like that users use only YouPHPTube setraming apllication and when they upload the video there it will be encoded automatically without user use of the encoding application. 

Is it possible?

Thanks.

**Comment by flochman** [comment]
I have setup encoder http to my local encoding application in advanced config on streamer application.

**Comment by flochman** [comment]
Aaach, I see how it works. It littlebit confusing. In right menu is Upload the plain video without encoding and also Encoding possibility which redirect you to encoding appplication.

OK. It seems that evrything is working, but the menu is littebit confusing for new users.

OK. Thanks.

**Comment by DanielnetoDotCom** [comment]
> ...I would like that users use only YouPHPTube setraming apllication and when they upload the video there it will be encoded automatically without user use of the encoding application.

To encode videos, you must use the encoder, there is no other option

> ...I have only one question. How to configure encoding application?

What do you mean? did you check this: https://tutorials.youphptube.com/video/streamer-and-encoder

**Comment by flochman** [comment]
Thanks.

I mean to config encoder application for example I want restrict user only code the Low or SD resolution. Or switchoff possiblity to Advanced menu like extract audio and so on.

Is also question if I can leave webrowser page when the encoding process is running. If I have many users with many videos to encode in queue, thay will not wait after their video wil be encoded on next day.

Thanks.

**Comment by DanielnetoDotCom** [comment]
> I mean to config encoder application for example I want restrict user only code the Low or SD resolution. Or switchoff possiblity to Advanced menu like extract audio and so on.

We do not have this yet, but I will probably make that change soon, so many users want this

> Is also question if I can leave webrowser page when the encoding process is running. If I have many users with many videos to encode in queue, thay will not wait after their video wil be encoded on next day.

you can leave the browser with no problem, the encoder process is on server background
